### DISCUSSION TITLE
Using Towny Advanced in -64 to 320 worlds.

### DISCUSSION THREAD

**Discussion by ghost** [TownyAdvanced/Towny#5052] [Q&A] [answered]
Hello, I am setting up a paper server with your TownyAdvanced plugin, I am using the data pack of the new caves as world generator, so my worlds are not 0 to 256, they are -64 to 320, I am going to have problems with the town blocks? Does it support negative and over 256 heights? I should use the plugin in a different world with only 0 to 256 heights?

**Comment by LlmDl** [comment, marked as answer]
Towny is mostly height agnostic, it shouldn't care since its all the same vertically. You should almost definitely make sure the revert-on-unclaim feature is turned off though.

**Reply by LlmDl** [reply]
I would be interested in knowing how it works in general, I wouldn't assume anything will go wrong.

**Reply by ghost** [reply]
Okay, thanks you a lot because of you fast and good answering.
